src/Commands/AppConfig/Account/AccountListCommand.cs

```diff
@@ -11,7 +11,7 @@ namespace AzureMcp.Commands.AppConfig.Account;
 
```

Comment:
Are these conventions coming from .NET sdk repo?  If not, let's take the time to snap to that.

I'm wondering why some private are _camel and some are Pascal.  

new-command.md will also need to be updated.

Reply:
These naming changes conflict with the ones in our editorconfig.
https://github.com/Azure/azure-mcp/blob/main/.editorconfig#L54-L57

We execute dotnet format before pushing any changes to GitHub: https://github.com/Azure/azure-mcp/blob/main/eng/scripts/git-hooks/pre-push#L15

dotnet format uses the rules in .editorconfig.